templates/gitserver/gitserver.StatefulSet.yaml

```diff
@@ -6,29 +6,25 @@
 {{- $_ := set $envVars "ORIGIN_MAP" $global.Values.site.gitOriginMap -}}
 {{- include "collectEnv" (list $envVars .Values.cluster.gitserver.containers.gitserver.env) -}}
 
-{{- range $index, $gitserverIndex := until (required ".Values.cluster.gitserver.shards required" .Values.cluster.gitserver.shards | int) }}
-{{- if eq $index 0 | not }}
----
-{{ end -}}
-apiVersion: extensions/v1beta1
-kind: Deployment
+apiVersion: apps/v1
```

Comment:
This requires Kubernetes 1.9+
If we used `apps/v1beta2` the minimum requirement would be 1.8+
If we used `apps/v1beta1` the minimum requirement would be 1.5+

I tested with `apps/v1beta1` on Kubernetes 1.8.10-gke.0 (which is the lowest GCP allows) and it seemed to work.

We should see what version of Kubernetes our customers are already using, choose an appropriate value here, and document the minimum version of Kubernetes that we require.

Reply:
Not sure how relevant all the above is, but those seem to be the changes that affect stateful sets

Reply:
We should prefer `apps/v1` unless we find out that one of our existing DC customers is using less that 1.9